btrdbextras/dq.py

```diff
@@ -29,9 +29,9 @@ def __init__(self, btrdb, uu):
         # stream name, so we will need to be careful how we name distillates
         types = re.findall(r"(?=("+'|'.join(KNOWN_DISTILLER_TYPES)+r"))", self.name)
         if len(types) == 0:
-            raise Exception(f"unknown distiller type. Must be one of [{', '.join(KNOWN_DISTILLER_TYPES)}]")
+            raise ValueError(f"unknown distiller type. Must be one of [{', '.join(KNOWN_DISTILLER_TYPES)}]")
         if len(types) > 1:
-            raise Exception(f"ambiguous distiller name. contains references to [{', '.join(types)}]")            
+            raise ValueError(f"ambiguous distiller name. contains references to [{', '.join(types)}]")
```

Comment:
Do we want to ignore the ambiguous distiller errors when creating a new DQStream? I feel like these might be useful to be notified of